### ISSUE TITLE
Support for service attribute in hosts criteria is broken

### ISSUE THREAD

**Issue by igarny** [perfsonar/meshconfig-admin#34]
I have tried directly one of the examples on this page http://docs.perfsonar.net/mca_userguide_hostgroup.html#dynamic-host-groups

`if(service.location.country == "US" || service.location.country == "CA") return true;`

This type of filter fails with: **perfsonar1.icepp.jp - ReferenceError: service is not defined**

Also please note it is important for GEANT to be able to assign different interfaces to the respective services. Meaning the hostname assignment for us is useless.

Regards,
Ivan

**Comment by soichih** [comment]
The documentation needs to be updated.

Something like this should work

```
if(host.info["location-country"] == "US" || host.info["location-country"] == "CA") return true;
```

> different interfaces to the respective services. Meaning the hostname assignment for us is useless.
I am not sure if I understand this, but I've seen a conversation about somethingrelated to this last month. @apertome

**Comment by apertome** [comment]
Fixed in perfsonar/docs@319331400f57c2d6e2a786e48b0739e2a81a5b1d
